src/ml_board/backend/dummy_server/dummy_server.py

```diff
@@ -151,7 +151,7 @@ def terminate(text:str) -> None:
         # run the Server
         WebSocketWrapper(
             flask_app=create_flask_server(),
-            host="localhost",
+            host="127.0.0.1",
```

Comment:
I made this change because I kept getting this error  `[Errno 11001] No address found`